android/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.3.1
+version: 1.3.2
 apiversion: 2
```

Comment:
Anything that's compiled with TiSDK 6+ would need a bump to apiversion 3. This is to differentiate them. TiSDK 6+ has an updated V8 for Android. They are totally not compatible with any modules compiled prior to TiSDK 6.